### PR TITLE
Application Connector updated introduction

### PR DESCRIPTION
**Pull request by tomekpapiernik**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**Description**

Changes proposed in this pull request:

- Updated introduction for the Application Connector component

### CODE REVIEW

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 1]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
...
 
-The Application Connector is a proprietary Kyma solution. This endpoint is the Kyma side of the connection between Kyma and external solutions. The Application Connector provides the following functionalities:
-* Metadata API for registering available external solutions' APIs, Event Catalog, and documentation
-* Events API for sending Events from the external solution to the Kyma Event Bus
-* Proxy API for calling registered external solutions' APIs from Kyma services or lambdas
+- The **Connector Service** generates the required certificates and ensures a secure and trusted connection between Kyma and an external solution.
+- The **Metadata API** allows you to register all of the external solution's APIs and Event catalogs which Kyma consumes.
+- The **Gateway Service** proxies the calls and Events sent between Kyma and the registered APIs an external solution, and routes the traffic to the appropriate components and APIs on both sides of the connection.
+
+To ensure maximum security and separation, a single instance of Application Connector allows you to connect only to a single external solution. This connection is represented in Kyma by a Remote Environment.
```
Comment (akgalwas) [inline docs/application-connector/docs/001-overview-application-connector.md]: I'm not sure whether the "a single instance of Application Connector" expression is correct in this context. I would rather use "Gateway" here. Both Metadata and Connector Service are singletons. There is one instance of Gateway per remote environment.

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 2]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
...
 ---
 
+The Application Connector is a proprietary Kyma component that allows you to connect with external solutions. The Application Connector consists of three
+components that ensure the security of the connection, the access to all of the external solution's Events and APIs, as well as proper routing of the calls and Events coming from an external solution to Kyma, and from Kyma to the connected external solution.
 
-The Application Connector is a proprietary Kyma solution. This endpoint is the Kyma side of the connection between Kyma and external solutions. The Application Connector provides the following functionalities:
-* Metadata API for registering available external solutions' APIs, Event Catalog, and documentation
-* Events API for sending Events from the external solution to the Kyma Event Bus
-* Proxy API for calling registered external solutions' APIs from Kyma services or lambdas
+- The **Connector Service** generates the required certificates and ensures a secure and trusted connection between Kyma and an external solution.
```
Comment (akgalwas) [inline docs/application-connector/docs/001-overview-application-connector.md]: We need to have more consistent naming here. 
My proposal is:
- Connector Service
- Metadata Service
- Gateway Service

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 3]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
 type: Overview
 ---
 
+The Application Connector is a proprietary Kyma implementation that allows you to connect with external solutions. The Application Connector consists of three
+components that ensure the security of the connection, the access to all of the external solution's Events and APIs, as well as proper routing of the calls and Events coming from an external solution to Kyma, and the API calls sent from Kyma to the connected external solution.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/001-overview-application-connector.md]: This sentence is so loooooong. Can you make a few shorter ones out of it?

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 4]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
...
 
-The Application Connector is a proprietary Kyma solution. This endpoint is the Kyma side of the connection between Kyma and external solutions. The Application Connector provides the following functionalities:
-* Metadata API for registering available external solutions' APIs, Event Catalog, and documentation
-* Events API for sending Events from the external solution to the Kyma Event Bus
-* Proxy API for calling registered external solutions' APIs from Kyma services or lambdas
+- The **Connector Service** generates the required certificates and ensures a secure and trusted connection between Kyma and an external solution.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/001-overview-application-connector.md]: I'd precede it with an introduction sentence. For example:
`These three components are as follows:`
What do you think?

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 5]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
...
+- The **Metadata Service** allows you to register all of the external solution's APIs and Event catalogs which Kyma consumes.
+- The **Gateway Service** proxies the calls and Events sent between Kyma and the registered APIs an external solution, and routes the traffic to the appropriate components and APIs on both sides of the connection.
+
+To ensure maximum security and separation, a single instance of the Gateway Service allows you to connect only to a single external solution. This connection is represented in Kyma by a Remote Environment.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/001-overview-application-connector.md]: Maybe a link to the RE documentation?

File: docs/application-connector/docs/001-overview-application-connector.md (hunk 4 above)
Comment (klaudiagrz) [inline docs/application-connector/docs/001-overview-application-connector.md]: "an external solution" doesn't fit here (as discussed)

File: docs/application-connector/docs/001-overview-application-connector.md [hunk 6]
```diff
@@ -3,8 +3,11 @@ title: The Application Connector
...
 
-The Application Connector is a proprietary Kyma solution. This endpoint is the Kyma side of the connection between Kyma and external solutions. The Application Connector provides the following functionalities:
-* Metadata API for registering available external solutions' APIs, Event Catalog, and documentation
-* Events API for sending Events from the external solution to the Kyma Event Bus
-* Proxy API for calling registered external solutions' APIs from Kyma services or lambdas
+- The **Connector Service** generates the required certificates and ensures a secure and trusted connection between Kyma and an external solution.
+- The **Metadata Service** allows you to register all of the external solution's APIs and Event catalogs which Kyma consumes.
+- The **Gateway Service** proxies the calls and Events sent between Kyma and the registered APIs an external solution, and routes the traffic to the appropriate components and APIs on both sides of the connection.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/001-overview-application-connector.md]: missing "of" as discussed :)

**Pull request merged**